### PR TITLE
replace logo, replace all black and dark grey with the same hex code

### PR DESCRIPTION
**Pull request by MariaNono**

### CODE REVIEW

File: css/normalize.css [hunk 1]
```diff
@@ -120,8 +120,7 @@ h1 {
  */
 mark {
   background: #ff0;
-  color: #000;
-}
+  color: #12121F;}
```
Comment (Smokyish) [inline css/normalize.css:123]: Needs a linebreak for `} `

File: css/trustlines-network.webflow.css [hunk 2]
```diff
@@ -160,8 +160,7 @@ ul {
   margin-top: 0px;
   margin-bottom: 10px;
   padding-left: 40px;
-  color: #000;
-}
+  color: #12121F;}
```
Comment (Smokyish) [inline css/trustlines-network.webflow.css:163]: Linebreak for `}`

File: css/trustlines-network.webflow.css [hunk 3]
```diff
@@ -902,8 +899,7 @@ blockquote {
 .column {
   padding-right: 0px;
   padding-left: 0px;
-  border-right: 1px none #000;
-}
+  border-right: 1px none #12121F;}
```
Comment (Smokyish) [inline css/trustlines-network.webflow.css:902]: Linebreak

File: css/trustlines-network.webflow.css [hunk 4]
```diff
@@ -3258,16 +3243,14 @@ blockquote {
...
+  color: #12121F;}
 
 .eco-description {
   width: auto;
   padding-right: 50px;
   padding-left: 50px;
   font-family: 'Inter';
-  color: #4b4b4b;
-  font-size: 16px;
+  color: #12121F;  font-size: 16px;
```
Comment (Smokyish) [inline css/trustlines-network.webflow.css:3253]: Linebreak needed for `font-size`

**Comment by Smokyish** [comment]
@andfletcher I have reviewed this PR and left a comment review above. I can't merge here so please have a look at this PR.

**Comment by andfletcher** [comment]
@MariaNono logo is too small. Compared to the old logo this just hurts my eyes looking at it since it is fuzzy and to small. Please find a way to fix this so that the new logo also works with the current pages. We can not have a setback on logo quality after updating the logo.

**Comment by andfletcher** [comment]
@Smokyish was your change request done / talked about?

**Comment by Smokyish** [comment]
@andfletcher Not that I'm aware of.

**Comment by MariaNono** [comment]
@Smokyish, I have changed the navbar items font weight to 500.
Regarding the logo image, Sash prepared all the logos deliberately with that empty space around and the dimensions and placement in my pull request were thought through and I can't see a problem with it.

**Comment by Smokyish** [comment]
@andfletcher Ready to be merged.

**Pull request merged**